### PR TITLE
[TW-1661] pr review groups

### PR DESCRIPTION
**Pull request by petewitcher-postman**
Adds user groups to the PR creation procedure.

[Beta docs](https://tw-1661-pr-review-groups.learning.postman-beta.com/docs/collaborating-in-postman/using-version-control/creating-pull-requests/)

### CODE REVIEW

**Review by mark-anthony-postman — APPROVED** [review]
LGTM!

**Review by jkonrath-postman — APPROVED** [review]
LGTM

File: src/pages/docs/collaborating-in-postman/using-version-control/creating-pull-requests.md [hunk 1]
```diff
@@ -52,20 +52,20 @@ When you have made the changes that you want to a forked collection or environme
...
 1. Select **Create Pull Request**.
 
     <img alt="Create Pull Request" src="https://assets.postman.com/postman-docs/v10/pull-request-overview-v10.jpg" width="350px"/>
 
 [Postman notifies the reviewers](/docs/collaborating-in-postman/using-version-control/reviewing-pull-requests/#reviewing-pull-requests) you selected about your pull request. You will receive a notification if the reviewers [comment on](/docs/collaborating-in-postman/using-version-control/reviewing-pull-requests/#adding-comments), [approve](/docs/collaborating-in-postman/using-version-control/reviewing-pull-requests/#approving-a-pull-request), or [merge](/docs/collaborating-in-postman/using-version-control/reviewing-pull-requests/#merging-changes-from-a-pull-request) the pull request.
 
-> A reviewer must have an Editor role for the element to merge changes. If you add a user who doesn't have Editor access as a reviewer, there will be a warning icon next to their name.
+> A reviewer or group must have an Editor role for the element to merge changes. If you add a user or group that doesn't have Editor access as a reviewer, there will be a warning icon next to the name.
```
Comment (sony-mathew) [inline src/pages/docs/collaborating-in-postman/using-version-control/creating-pull-requests.md:62]: Hey @petewitcher-postman, Should we also say that these users who has view access can actually approve the PR, but can't merge them.?

Comment (petewitcher-postman) [reply src/pages/docs/collaborating-in-postman/using-version-control/creating-pull-requests.md]: Added a sentence at the end:

A reviewer or group must have the Editor role for the element to merge changes. If you add a user or group that doesn't have Editor access as a reviewer, there will be a warning icon next to the name. Reviewers and groups with the Viewer role can approve PRs, but can't merge PRs.

**Review by sony-mathew — APPROVED** [review]
Looks good.

**Pull request merged**